utils/results/conversion/simplejsonparser/simplejsonparser.go

```diff
@@ -157,6 +157,25 @@ func (sjc *CmdResultsSimpleJsonConverter) ParseSecrets(_ results.ScanTarget, isV
 	return
 }
 
+func (sjc *CmdResultsSimpleJsonConverter) ParseMalicious(_ results.ScanTarget, isViolationsResults bool, maliciousFindings []results.ScanResult[[]*sarif.Run]) (err error) {
+	if !sjc.entitledForJas {
+		return
+	}
+	if sjc.current == nil {
+		return results.ErrResetConvertor
+	}
+	maliciousSimpleJson, err := PrepareSimpleJsonJasIssues(sjc.entitledForJas, sjc.pretty, results.ScanResultsToRuns(maliciousFindings)...)
+	if err != nil || len(maliciousSimpleJson) == 0 {
+		return
+	}
```

Comment:
```suggestion
	if sjc.current == nil {
		return results.ErrResetConvertor
	}
    for i := range maliciousFindings {
		if shouldUpdateStatus(sjc.current.Statuses.MaliciousStatusCode, &maliciousFindings[i].StatusCode) {
			sjc.current.Statuses.MaliciousStatusCode = &maliciousFindings[i].StatusCode
		}
	}
	maliciousSimpleJson, err := PrepareSimpleJsonJasIssues(sjc.entitledForJas, sjc.pretty, results.ScanResultsToRuns(maliciousFindings)...)
	if err != nil || len(maliciousSimpleJson) == 0 {
		return
	}
```

Add this scan to the status attributes so we can use it in all other places to display the command information

Reply:
ok